flux.js

```diff
@@ -0,0 +1,26 @@
+export default function createStore(reducer, state) {
```

Comment:
We already have this file. 
Fix ways to it (in `src`)

Reply:
also done (hope it is corrected right)